src/components/Eth2Diagram.js

```diff
@@ -1,82 +1,185 @@
-import React, { useState } from "react"
+import React, { useState, useContext } from "react"
 import styled from "styled-components"
 import Modal from "./Modal"
 import ButtonLink from "./ButtonLink"
 import Link from "./Link"
+import { ThemeContext } from "styled-components"
+import Emoji from "./Emoji"
 
 const Eth1 = styled.div`
-  border: 1px solid ${(props) => props.theme.colors.black50};
-  background: ${(props) => props.theme.colors.success200};
+  cursor: pointer;
+  border: 1px solid
```

Comment:
Just to clarify my other point re: not passing in `isDarkTheme` as a prop... we should create a color variable in theme.js (e.g. `mainnetBorder` that is `success300` for dark mode & `black50` for light mode). That way we avoid the need of passing the props around & this conditional logic based off it.